Dockerfile.jenkins

```diff
@@ -38,72 +38,13 @@ RUN if [ -f /usr/bin/gcov-dump-$gccvers ] ; then \
 RUN if [ -f /usr/bin/gcov-tool-$gccvers ] ; then \
       update-alternatives --install /usr/bin/gcov-tool gcov-tool /usr/bin/gcov-tool-$gccvers 10 ; \
     fi
-
-# Set up for Miniconda
-WORKDIR /tmp
-RUN wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-RUN bash miniconda.sh -b -p /usr/local/miniconda
-ENV PATH /usr/local/miniconda/bin:$PATH
+    
+# Set up conda
+RUN wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh
+RUN bash /tmp/miniconda.sh -b -p /usr/local/miniconda
+ENV PATH=/usr/local/miniconda/bin:$PATH
 RUN conda config --set always_yes yes --set changeps1 no
 RUN conda update -q conda
-RUN ln -s /usr/local/miniconda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
```

Comment:
I'm trusting you here

Reply:
as in, for all these deletions here

Reply:
The deletions are kind of spurious in the context of this PR - it's in the nature of a 'complete rewrite' as opposed to a modification, but diff does its best to generate a set of changes leaving some of the initial version intact ... :-)